code/modules/mob/living/carbon/human/species_types/synths.dm

```diff
@@ -1,37 +1,86 @@
-/datum/species/synth
-	name = "Synth" //inherited from the real species, for health scanners and things
+#define isipcperson(A) (is_species(A, /datum/species/synth))

```

Comment:
Put this in with the other istype helpers in `code/__DEFINES/is_helpers.dm`. Also just call it `isipc(`